.github/workflows/deploy-test.yml

```
@@ -0,0 +1,36 @@
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      NPM_AUTH_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}
+      SLACK_INCOMING_HOOK: ${{ secrets.SLACK_INCOMING_HOOK }}
+    steps:
+      - name: Checkout target repo
+        uses: actions/checkout@v4
+        with:
+          repository: uploadcare/react-file-uploader
+          path: react-file-uploader
+
+      - name: Update uploader version in monorepo
+        run: |
```

actionlint (1.7.4)

property "get-version" is not defined in object type {}

(expression)